components/messaging/CurrentConversation.tsx

```diff
@@ -86,6 +108,20 @@ const CurrentConversation: React.FC = () => {
 
   return (
     <div className='conversation-height mb-10 flex flex-1 flex-col justify-between bg-[#fafaf9] shadow-inner'>
+      <div className='flex flex-row p-5'>
+        <p data-testid='item-donor'>
+          <b>From: </b>
+          {itemDonor && itemDonor.username}
+        </p>
+        {itemDonor && (
+          <img
+            className='ml-2'
+            alt='user logo'
+            width='25'
+            src={itemDonor.avatar}
```

Comment:
If a user has not set a profile image this value will return as `null`, which then leaves the page showing an ugly img placeholder.

This is potentially a bigger issue to solve in how the profile details are being saved but could also maybe be solved here with a nullish coalescent operator"

```js
src={itemDonor.avatar ?? '/default-profile.png'}
```
This will apply the itemDonor.avatar value unless it is nullish, in which case it will use the route to the default image we currently use in the `/profile` page.

Reply:
Saw this added to the recent changes, thanks! 

Can you add the classes `rounded-full` to the `Image` element and `items-center` to the flexbox `div`?

Those two combined will mimic the look on the PR description:
![Screenshot 2024-03-13 at 14 11 42](https://github.com/enBloc-org/kindly/assets/114600712/64c09064-ef03-43f7-86a5-6295a1880d50)